fe/src/main/java/org/apache/doris/alter/MaterializedViewHandler.java

```diff
@@ -858,4 +1084,56 @@ public void cancel(CancelStmt stmt) throws DdlException {
             jobDone(rollupJob);
         }
     }
+
+    // There may be dependency relation between user batch input rollup index
+    // e.g, user input rollup index: r1 from r2,r2 from r3 ;the input order must be r3,r2,r1
+    // the reason for sort rollup index:
+    //   son rollup must be dealed after parent rollup, because they need parent rollup's info,such as baseRollupIndex
+    public static List<AlterClause> sortRollupIndex(List<AlterClause> alterClauseList) {
```

Comment:
This is unnecessary. We could check the
`alter statement `and **require the from index must exist firstly**, and then, we needn't do this work

Reply:
👌